src/vsr/replica.zig

```diff
@@ -324,7 +324,6 @@ pub fn ReplicaType(
         /// Seeded with the replica's index number.
         prng: std.rand.DefaultPrng,
 
-        context: ?*anyopaque = null,
```

Comment:
I think `cluster.zig` uses this as an anchor for `@fieldParentPtr` to turn a `*Replica` back into a `*Cluster`.

Reply:
Indeed, added a comment instead (it's _not_ `@fieldParentPtr`, so that's why we need context).